src/winservices.go

```diff
@@ -120,10 +120,8 @@ func run(e *exporter.Exporter, i *integration.Integration, interval time.Duratio
 			if err != nil {
 				log.Error("failed to publish integration:%v", err)
 			}
-			log.Debug("Metrics and inventory published")
```

Comment:
I would not remove debug logs in case we need them in the future to help users.
Do you think they are too many/ too verbose?

Reply:
I agree but just removed these only two since both have logs printed immediately after or before.  In this particular case, the published json should be printed or the fail to publish error

Reply:
make sense!